data/json/monsterdrops/feral_humans.json

```diff
@@ -800,11 +800,10 @@
           {
```

Comment:
no need for this to be nested in a distribution, since it is by itself

Reply:
ah ty.